src/System Application/App/AppSource Gallery/src/IAppSourceProductManagerDependencies.Interface.al

```diff
@@ -0,0 +1,38 @@
+
+// ------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// ------------------------------------------------------------------------------------------------
+namespace System.Apps.AppSource;
+
+using System.Environment.Configuration;
+using System.RestClient;
+
+/// <summary>
+/// Interface for managing dependencies to the AppSource Product Manager codeunit.
+/// </summary>
+interface "IAppSource Product Manager Dependencies"
```

Comment:
We do not I our interfaces the same as we do not C our codeunits or P our pages... Just drop the I :-)

Reply:
Next PR